etc/openqa/workers.ini

```diff
@@ -5,7 +5,10 @@
 [global]
 # space separated list of webuis to connect to (empty defaults to localhost)
 #HOST = http://openqa.example.host
+HOST = http://localhost:9526
 
+# Ensure that Multimachine jobs are scheduled on the same worker host.
+#PARALLEL_ONE_HOST_ONLY = 1
```

Comment:
```suggestion
# Enable to restrict any multi-machine jobs taken by this worker host to only run here.
# Disabled by default.
#PARALLEL_ONE_HOST_ONLY = 0
```

Reply:
```suggestion
# Enable to restrict any multi-machine jobs taken by this worker host to only run there.
# Disabled by default.
#PARALLEL_ONE_HOST_ONLY = 0
```

Reply:
None of our suggestions has been applied.